### ISSUE TITLE
Deployment

### ISSUE THREAD

**Issue by AndrewKvalheim** [SeaGL/attend.seagl.org#3]
Automate deployment from the `production` branch to `https://attend.seagl.org/`.

**Comment by strugee** [comment]
Any reason we can't just deploy this to GitHub Pages instead of CloudFront?

**Comment by AndrewKvalheim** [comment]
Nope, that should work fine. In 2020 when I started this S3 was just the solution I was most familiar with and the repository [wasn’t even on GitHub](https://github.com/SeaGL/organization/issues/227). I still haven’t read much about GitHub Actions and still know GitHub Pages only as a Jekyll host.
